### PR TITLE
feat: use glibc toolchain for nvidia

### PR DESCRIPTION
**Pull request by frezbo**
Use wolfi base as toolchain for NVIDIA build.
This removes a lot of hacks and patches we maintain.

Fixes: #171
Fixes: https://github.com/siderolabs/pkgs/issues/720

### CODE REVIEW

**Review by smira — APPROVED** [review]

File: logging/syslog/syslog.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# https://download.nvidia.com/XFree86/Linux-x86_64/515.65.01/README/nvidia-persistenced.html
```
[the remark this answers is not in the document]
Comment (frezbo) [reply logging/syslog/syslog.yaml:1]: oh I'll remove that, syslog was for debugging

File: nvidia-gpu/nvidia-container-toolkit/nvidia-persistenced-wrapper/main.go [hunk 2]
```diff
@@ -41,11 +41,7 @@ func main() {
 		}
 	}
 
-	cmd := exec.Command("/usr/local/bin/nvidia-persistenced",
-		[]string{
-			"--no-persistence-mode",
-			"--verbose",
-		}...)
+	cmd := exec.Command("/usr/local/bin/nvidia-persistenced")
```
[the remark this answers is not in the document]
Comment (frezbo) [reply nvidia-gpu/nvidia-container-toolkit/nvidia-persistenced-wrapper/main.go:44]: yes, the persistenced process doesn't have no daemon option, so it exits without the wrapper

[the remark this answers is not in the document]
Comment (smira) [reply nvidia-gpu/nvidia-container-toolkit/nvidia-persistenced-wrapper/main.go:44]: ah, it daemonizes itself, got it.

**Pull request merged**